lib/dotcom_web/components/system_status/widget.ex

```diff
@@ -0,0 +1,135 @@
+defmodule DotcomWeb.Components.SystemStatus.Widget do
+  @moduledoc """
+  A component that renders the given `@statuses` in a table.
+  """
+
+  use DotcomWeb, :component
+
+  import DotcomWeb.Components.RoutePills
+  import DotcomWeb.Components.SystemStatus.StatusLabel
+
+  @route_ids ["Red", "Orange", "Green", "Blue"]
+
+  def system_status_widget(assigns) do
+    assigns = assigns |> assign(:rows, status_to_rows(assigns.routes_with_statuses))
+
+    ~H"""
+    <div class="px-5 py-4 border-[1px] border-gray-lightest rounded-lg w-96">
+      <div class="border-gray-lightest border-b-[1px] pl-2 pb-3 flex items-center gap-2">
+        <.icon type="icon-svg" name="icon-mode-subway-default" class="h-7 w-7" />
+        <span class="font-heading font-bold text-[1.75rem]">Subway Status</span>
+      </div>
+      <div
+        :for={row <- @rows}
+        class={[
+          "flex gap-2",
+          !row.style.short_bottom_border && "border-gray-lightest border-b-[1px]",
+          "hover:bg-brand-primary-lightest cursor-pointer group/row"
+        ]}
+      >
+        <div class={["pl-2 py-3", row.style.hide_route_pill && "invisible"]}>
+          <.route_pill_with_modifiers
+            route_id={row.route_info.route_id}
+            modifier_ids={row.route_info.branch_ids}
+            modifier_class="group-hover/row:ring-brand-primary-lightest"
+          />
+        </div>
+        <div class={[
+          "py-3 w-full",
+          row.style.short_bottom_border && "border-b-[1px] border-gray-lightest"
+        ]}>
+          <.status_label
+            status={row.status_entry.status}
+            prefix={row.status_entry.prefix}
+            plural={row.status_entry.plural}
+          />
+        </div>
+      </div>
+    </div>
+    """
+  end
+
+  defp status_to_rows(data) do
```

Comment:
Happy to add documentation-y comments for these private functions, but I wanted to punt that until just before merging.